Bigtable/src/Admin/V2/Gapic/BigtableInstanceAdminGapicClient.php

```diff
@@ -1416,70 +1511,74 @@ public function updateAppProfile($appProfile, $updateMask, array $optionalArgs =
-     *          cluster. Currently only zones are supported, so values should be of the
-     *          form `projects/{project}/locations/{zone}`.
+     *           (`CreationOnly`)
+     *           The location where this cluster's nodes and storage reside. For best
+     *           performance, clients should be located as close as possible to this
+     *           cluster. Currently only zones are supported, so values should be of the
+     *           form `projects/{project}/locations/{zone}`.
      *     @type int $state
-     *          The current state of the cluster.
-     *          For allowed values, use constants defined on {@see \Google\Cloud\Bigtable\Admin\V2\Cluster\State}
+     *           The current state of the cluster.
+     *           For allowed values, use constants defined on {@see \Google\Cloud\Bigtable\Admin\V2\Cluster\State}
+     *     @type int $serveNodes
+     *           The number of nodes allocated to this cluster. More nodes enable higher
+     *           throughput and more consistent performance.
+     *     @type ClusterConfig $clusterConfig
+     *           Configuration for this cluster.
      *     @type int $defaultStorageType
-     *          (`CreationOnly`)
-     *          The type of storage used by this cluster to serve its
-     *          parent instance's tables, unless explicitly overridden.
-     *          For allowed values, use constants defined on {@see \Google\Cloud\Bigtable\Admin\V2\StorageType}
+     *           (`CreationOnly`)
+     *           The type of storage used by this cluster to serve its
+     *           parent instance's tables, unless explicitly overridden.
+     *           For allowed values, use constants defined on {@see \Google\Cloud\Bigtable\Admin\V2\StorageType}
      *     @type EncryptionConfig $encryptionConfig
-     *          Immutable. The encryption configuration for CMEK-protected clusters.
+     *           Immutable. The encryption configuration for CMEK-protected clusters.
      *     @type RetrySettings|array $retrySettings
-     *          Retry settings to use for this call. Can be a
-     *          {@see Google\ApiCore\RetrySettings} object, or an associative array
-     *          of retry settings parameters. See the documentation on
-     *          {@see Google\ApiCore\RetrySettings} for example usage.
+     *           Retry settings to use for this call. Can be a
+     *           {@see Google\ApiCore\RetrySettings} object, or an associative array of retry
+     *           settings parameters. See the documentation on
+     *           {@see Google\ApiCore\RetrySettings} for example usage.
      * }
      *
      * @return \Google\ApiCore\OperationResponse
      *
      * @throws ApiException if the remote call fails
-     * @experimental
      */
-    public function updateCluster($name, $serveNodes, array $optionalArgs = [])
+    public function updateCluster($name, array $optionalArgs = [])
```

Comment:
This is BC-breaking, we missed it in the last GAPIC release (I used `server_nodes` instead of `serve_nodes`)

Fixed in https://github.com/googleapis/gapic-generator-php/pull/449/files